openapi/index.yaml

```
@@ -46,9 +45,34 @@ paths:
           description: Generic server error
           schema:
             $ref: '#/definitions/ProblemJson'
+  /organizations/{organization_fiscal_code}:
```

The resource here is _the list of **delegates** for an organization_, maybe is better to make it clear in the url so that it's not mistaken with _the detail of an organization_.

```suggestion
  /organizations/{organization_fiscal_code}/delegates:
```

---

Yes, it's done by 7c37e00